Assets/Shaders/Excellent/Excellent Icosahedron/Excellent Icosahedron.cginc

```diff
@@ -0,0 +1,188 @@
+}
+
+float sdStarPrism(float3 objectPos, float scale)
+{
+    objectPos.xy = pModRotate(objectPos.xy, UNITY_HALF_PI);
+    objectPos.xy = pModPolar(objectPos.xy, 5.0);
+    objectPos -= float3(0.1, 0.0, 0.0);
+    objectPos.xy = pModRotate(objectPos.xy, -UNITY_HALF_PI);
+
+    objectPos /= scale;
+    return sdTriPrism(objectPos, float2(0.1, 0.25)) * scale;
+}
+
+
+void ELBoundingBox(out float3 boxMin, out float3 boxMax)
+{
+    boxMin = float3(-_Bound, -_Bound, -_Bound);
+    boxMax = float3( _Bound,  _Bound,  _Bound);
+}
+
+// Slightly larger geometry cube to encompass the entire raymarch.
+[maxvertexcount(24)]
+void ExcellentGeometryCube(line ELRaycastBaseVertexInput input[2], inout TriangleStream<ELRaycastBaseFragmentInput> triStream)
+{
+    static const float4 cv[8] = { float4(-_Bound, -_Bound, -_Bound, 1.0),
+                                  float4(-_Bound, -_Bound,  _Bound, 1.0),
+                                  float4(-_Bound,  _Bound, -_Bound, 1.0),
+                                  float4(-_Bound,  _Bound,  _Bound, 1.0),
+                                  float4( _Bound, -_Bound, -_Bound, 1.0),
+                                  float4( _Bound, -_Bound,  _Bound, 1.0),
+                                  float4( _Bound,  _Bound, -_Bound, 1.0),
+                                  float4( _Bound,  _Bound,  _Bound, 1.0) };
+    static const float t = 0.57735026916;
+    static const float4 cn[8] = { float4(-t, -t, -t, 1.0),
+                                  float4(-t, -t,  t, 1.0),
+                                  float4(-t,  t, -t, 1.0),
+                                  float4(-t,  t,  t, 1.0),
+                                  float4( t, -t, -t, 1.0),
+                                  float4( t, -t,  t, 1.0),
+                                  float4( t,  t, -t, 1.0),
+                                  float4( t,  t,  t, 1.0) };
+
+    static const uint cf[24] = { 0, 1, 2, 3,    // left
+                                 0, 2, 4, 6,    // front  
+                                 4, 6, 5, 7,    // right
+                                 7, 3, 5, 1,    // back
+                                 2, 3, 6, 7,    // top
+                                 0, 4, 1, 5  }; // bottom
+
+    ELRaycastBaseVertexInput output = input[0];
+    for (int i = 0; i < 6; i++)
+    {
+        for (int j = 0; j < 4; j++)
+        {
+            float vi = cf[i * 4 + j];
+            output.objectPos = cv[vi];
+            output.objectNormal = cn[vi];
+            triStream.Append(ELRaycastBaseVertex(output));
+        }
+        triStream.RestartStrip();
+    }
+}
+
+
+
+float2 ELMap(float3 objectPos)
+{
+    float rotation = _Time[1] * 25.0;
+
+    objectPos = ELRotateAroundYInDegrees(objectPos, rotation);
+	float t = _Time.x;
+	float a = 3 * UNITY_PI * t;
+	float s = pow(sin(a), 2.0);
+	float d1 = sdIcosahedron2(objectPos, 1.00);
+    float d2 = sdRoundBox9PatchField(
+        objectPos,
+        0.1 - 0.1 * s,
+        0.1 / length(objectPos * 2.),
```

Comment:
The compiler may see this one already, but `a/length(v b)` should be the same as saying `a/b length(v)` which is then the same as `(a/b) / length(v)` - i.e., this expression can be simplified to `0.05 / length(objectPos)` and there is one less multiplication to perform.